.travis.yml

```diff
@@ -1,15 +0,0 @@
----
-dist: trusty
```

Comment:
I don't think we need travis anymore because we aren't generating the html via doctoolchain.